app/models/event.rb

```diff
@@ -0,0 +1,9 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: :User, foreign_key: :creator_id
+  has_many :event_attendances, foreign_key: :attended_event_id
+  has_many :attendees, through: :event_attendances, source: :attendee
+  has_one_attached :main_image
+
+  scope :past, -> { where('event_date < ?', Time.now).order('event_date DESC') }
+  scope :upcoming, -> { where('event_date > ?', Time.now).order('event_date DESC') }
```

Comment:
Use these scopes in your `@past_events` and `@upcoming_events` in your Event Controller's show action.

Reply:
These scopes are created for Past events and Upcoming event for Event Model. Which we can navigate through navbar tabs.

![Screenshot 2020-10-22 at 6 44 09 PM](https://user-images.githubusercontent.com/47393794/97022712-12c09c80-1572-11eb-8503-15c793f2a0f3.png)

![Screenshot 2020-10-22 at 6 44 01 PM](https://user-images.githubusercontent.com/47393794/97022731-1c4a0480-1572-11eb-96bf-1221c3a9da08.png)